src/backend/gl/src/queue.rs

```diff
@@ -227,11 +233,19 @@ impl CommandQueue {
             .framebuffer_object;
 
         unsafe {
+            let tmp_read_fbo = gl.context.create_framebuffer().expect("TODO");
```

Comment:
Problem is that this code doesn't really use `tmp_read_fbo` anywhere. All it does is creating it and then destroying it. Perhaps, you forgot to fix the next line with `bind_framebuffer`?

Reply:
Ah yes! Whoops.